README.md

```diff
@@ -1,488 +1,57 @@
-
-    In summary, in order to make `go test` work with JWT you will need to set the following env vars:
-
-    * `TOKEN_GENERATOR_PRIVATE_RSA256_KEY`
-    * `TEST_RSA256_PUBLIC_KEY`
-    * `IGN_TEST_JWT`
-
-
-# Run the backend server
-
-First, make sure to set the `AUTH0_RSA256_PUBLIC_KEY` environment variable with the Auth0 RSA256 public key. This env var will be used by the backend to decode and validate any received Auth0 JWT tokens.
-Note: You can get this key from: <https://osrf.auth0.com/.well-known/jwks.json> (or from your own auth0 user). Open that url in the browser and copy the value of the `x5c` field.
-
-```
-$GOPATH/bin/web-cloudsim
-```
-
-
-## Cloudsim optional Env vars (in the .env file)
-
-- export IGN_CLOUDSIM_SSL_PORT=(default 4431)
-- export IGN_CLOUDSIM_HTTP_PORT=(default 8001)
-
-Also, if using Kubernetes with EC2:
-
-- export KUBEADM_JOIN=kubeadm join IP:PORT --token ...
-- export KUBE_MASTER_IP=the ip of the master node.
-- export IGN_TRANSPORT_IP_INTERFACE=`ethwe` (if using Weave Network in the kubernetes cluster)
-- export AWS_GZ_LOGS_BUCKET=a bucket name to upload gazebo logfiles.
-
-Note: To backup GZ logs into S3 it will also require a `kubernetes secret` called `aws-secrets` in the cluster.
-That secret should at least contain the following `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` entries.
-You can create the secret by running the script `create-secrets.sh` included with the web-cloudsim code.
-These are the contents of that script:
-
-```
-#/bin/bash
-
-# This script is used to create a Kubernetes secret with name 'aws-secrets' based on existing
-# environment variables: AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY.
-# This Secret will be used by the cloudsim server.
-
-kubectl create secret generic aws-secrets --from-literal=aws-access-key-id=${AWS_ACCESS_KEY_ID} --from-literal=aws-secret-access-key=${AWS_SECRET_ACCESS_KEY}
-```
-
-
-Note2: to find the value for the KUBEADM_JOIN env variable, from the kubernetes master, run:
-`sudo kubeadm token create --ttl 0 --print-join-command` and copy the results.
-Read the k8-readme.md for more info.
-
-
-
-# Kubernates
-The cloudsim server will try to connect to a Kubernates master. To do this, it will look
-for the following config file: `<home>/.kube/config`.
-
-
-For more details and tips about kubernetes, see the `k8-readme.md` file.
-
-
-
-# Linter
-
-1. Get the linter (gometalinter)
-
-    ```
-    cd ~/go_ws
-    ```
+[![Format](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/format.yaml?branch=main&label=Format)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Format")
+[![Test](https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/test.yaml?branch=main&label=Test)](https://github.com/gazebo-web/cloudsim/actions?query=workflow:"Test")
```

Comment:
Is this meant to reflect the latest status of the CI pipeline on `main`? If that is the case, I don't know how useful this is.

Reply:
What information does it intend to convey? Can you share some repos that use this so that I can check them ou?

Reply:
I stand corrected, I thought this was done in the same way in other repos, but that's not actually the case. We can remove the `branch` query param.

The problem with doing that is that if you access this link: https://img.shields.io/github/actions/workflow/status/gazebo-web/cloudsim/format.yaml?label=Format

you'll see that is marked as `passing`, but what branch is it referring to?

Reply:
Why don't we remove this? We can always check the state of the CI in the branch's commit.

Reply:
Sure!